eos-update-flatpak-repos

```diff
@@ -276,6 +277,49 @@ def write_deploy_file(deploy_file_path, orig_variant, idx, new_val):
                            new_variant.get_data_as_bytes().get_data())
 
 
+def mangle_firefox_metadata(metadata):
+    """Endless's Firefox Flatpak stored its data in the real ~/.mozilla in the user's
+    home directory. Mozilla's Firefox Flatpak stores its data in what it thinks is
+    ~/.mozilla, but which is actually ~/.var/app/$APP_ID/.mozilla through the magic of
+    --persist=.mozilla. Monkeypatch our Flatpak to match.
+
+    eos-migrate-firefox-profile is run via /etc/xdg/autostart to actually move the
+    data."""
```

Comment:
Is that true? I thought it was handled by the systemd user session.

Reply:
Great point. I was going to use xdg autostart but then decided to use systemd instead. In hindsight I'm not sure this was a good choice because of all the extra build system plumbing needed, but being able to use `systemctl status --user` was nice when debugging.

Reply:
I'm super pro-systemd and all the robust control over the execution environment, so for me it's clearly better than an autostart desktop file.